cmd/prober/rekor_v2.go

```diff
@@ -0,0 +1,102 @@
+// Copyright 2025 The Sigstore Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"net/http"
+	"strconv"
+
+	retryablehttp "github.com/hashicorp/go-retryablehttp"
+	"github.com/transparency-dev/tessera/api/layout"
+)
+
+// checkRekorV2CompleteFirstTile checks to see if the first tile in a shard is still a partial tile.
+// If no, then we must fetch a partial tile. If yes, we must fetch a full tile.
+// See https://github.com/C2SP/C2SP/issues/145.
+func checkRekorV2CompleteFirstTile(rekorURL string) (bool, error) {
```

Comment:
I think it would be better to just read the latest tile, not the first. The first is going to stay static forever, so this is just checking that that object is still in the bucket. Checking the latest actually gives a sense that Rekor is still publishing entries.

Reply:
changed to fetch the last tile, given a log size from the fetched checkoint.